modules/storage-persistent-storage-pv.adoc

```diff
@@ -163,7 +168,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin,microshift[]
 |NFS  | ✅ | ✅ | ✅
 |OpenStack Manila  | - | - | ✅
 |{rh-storage-first}  | ✅ | - | ✅
-|VMware vSphere | ✅ | - | ✅ ^[3]^ 
+|VMware vSphere | ✅ | - | ✅ ^[3]^
```

Comment:
This table isn't accurate to MicroShift.  In theory, the only item on this list should be LVMS, hostpath (the legacy plugin built into OpenShift/k8s, not the CSI plugin), and local volume, all of which only support RWO.

LVMS is the only CSI plugin MicroShift supports, while hostpath and local volume exist within OpenShift itself and not as separate plugins.